### PR TITLE
CSE-1087: Added option for phantom dependency analysis

### PR DESCRIPTION
**Pull request by dannykim-endor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ The following input parameters are also supported for the Endor Labs GitHub acti
 | Flags | Description |
 | :-- | :-- |
 | `additional_args` | Use additional_args to add custom arguments to the endorctl scan command. |
+| `disable_phantom` | Set to `false` to enable phantom dependency analysis. (Default: `true`) |
```
Comment (ewokcami) [inline README.md]: The double negative is confusing. Can we please change this to `phantom_dependencies` with the default being `false`

Comment (dannykim-endor) [reply README.md]: @ewokcami updated the flag name in the monorepo to be more clear. Once that is approved and merged, will update it here as well.

**Comment by dannykim-endor** [comment]
@dkourkouzelis please take another look

**Comment by dannykim-endor** [comment]
@dkourkouzelis can you take another look? Thanks!

File: README.md [hunk 2]
```diff
@@ -114,6 +114,7 @@ The following input parameters are also supported for the Endor Labs GitHub acti
 | Flags | Description |
 | :-- | :-- |
 | `additional_args` | Use additional_args to add custom arguments to the endorctl scan command. |
+| `phantom-dependencies` | Set to `true` to enable phantom dependency analysis. (Default: `false`) |
```
Comment (dkourkouzelis) [inline README.md]:
```suggestion
| `phantom_dependencies` | Set to `true` to enable phantom dependency analysis. (Default: `false`) |
```

**Review by dkourkouzelis — APPROVED** [review]
Looking good!

**Pull request merged**